active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,677 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Cleanlab to Determine Examples to Collect Additional Labels for in Data Labeled by Multiple Annotators to Improve Classification Models in Active Learning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how to compute an active learning score for every datapoint in your dataset, and shows how a classification model improves after relabeling examples for multiple rounds. The examples in the dataset will have been annotated by multiple annotators, and not all examples have to be labeled at the beginning. The computed active learning score will indicate if you should collect more labels for an already-labeled example (thus having a larger number of annotators label a potentially difficult example), or get a label for an example that has not been labeled.\n",
```

Comment:
```suggestion
    "This notebook demonstrates a practical approach to active learning for training  classification models with cleanlab. In active learning, we aim to construct a labeled dataset by collecting the fewest labels that still allow us to train an accurate classifier model. Here we assume data labeling is done in **batches**, and between these data labeling rounds, we retrain our classifier and combine it with CROWDLAB to decide what examples (i.e. datapoints) to label next round. We assume there are **multiple** annotators to label the data, but since their labels are not necessarily perfect, we sometimes wish to ask a new annotator to provide an extra label for a previously labeled example. This active learning setup is more widely applicable than academic studies which only consider labeling one example in each round or being limited to a single annotation per example.\n",
"\n",
 "cleanlab provides two useful estimates for multi-annotator active learning: (1) an active learning score quantifying how desirable it is to collect an additional label for every possible example, (2) accurate consensus labels for the previously labeled examples  that can be used for classifier model training.
This notebook demonstrates how to compute these easily for use in sequential active learning, showing how a classification model iteratively improves after (re)labeling examples for multiple rounds. Some examples in the dataset will have been labeled by multiple annotators, and not all examples have to have been labeled at the beginning. cleanlab's active learning score indicates if you should collect more labels for an already-labeled example (thus having a larger number of annotators label a potentially difficult example), or get a label for a potentially informative example that has not yet been labeled.\n",
```